blocks/carousel/carousel.css

```diff
@@ -37,22 +37,33 @@
 }
 
 .carousel .carousel-slide > div {
+    flex-direction: column;
     font-family: var(--serif-font);
     font-weight: var(--font-weight-light);
     line-height: var(--line-height-120);
     letter-spacing: var(--letter-spacing-1-75);
     font-size: var(--font-size-28);
 }
 
-.carousel .carousel-slide > figure {
```

Comment:
Since we are creating a new variation of the carousel special for "testimonial", instead of removing/changing the code under ` .carousel .carousel-slide ...`, please add your new code under new groupings specific to the testimonial class, i.e. `.carousel.testimonial .carousel-slide ...` .
Let me know if you have questions, I should have been more specific in my instructions.